check_source.py

```diff
@@ -350,6 +350,10 @@ def is_devel_project(self, source_project, target_project):
             return True
 
         # Allow any projects already used as devel projects for other packages.
```

Comment:
so this comment was for the section below, can you please keep it there?

Reply:
This section is amended for Factory hence I put it above.

Reply:
Just to add, the new code is "special handling" of new Factory as we transition this mechanic to git. The others uses (if any) are not affected here.